packages/eslint-plugin-office-addins/src/rules/no-office-initialize.ts

```diff
@@ -5,13 +5,17 @@ export = {
   meta: {
     type: <"problem" | "suggestion" | "layout">"suggestion",
     messages: {
-      noOfficeInitialize: "Office.onReady() is preferred over Office.initialize",
+      noOfficeInitialize:
+        "Office.onReady() is preferred over Office.initialize",
     },
     docs: {
       description: "Office.onReady() is more flexible than Office.initalize",
-      category: <"Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors">"Best Practices",
+      category: <
+        "Best Practices" | "Stylistic Issues" | "Variables" | "Possible Errors"
+      >"Best Practices",
       recommended: <false | "error" | "warn">false,
```

Comment:
Shouldn't this follow the same formatting as the cattegory?

Reply:
Not sure what you mean . . . this is the category.  Running prettier is what created this formatting.